README.md

```diff
@@ -44,10 +44,19 @@ The bot will provide you with help by asking it for `help`:
 
 [Issue Tracker](https://github.com/finos-fdx/bot-github-chatops/issues)
 
-## License
+## Contributing
+
+1. Fork it (<https://github.com/finos-fdx/bot-github-chatops/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](.github/CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
+4. Commit your changes (`git commit -am 'Add some fooBar'`)
+5. Push to the branch (`git push origin feature/fooBar`)
+6. Create a new Pull Request
 
-Copyright 2017 Fintech Open Source Foundation
+## License
 
-Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+The code in this repository is distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
 
 SPDX-License-Identifier: [Apache-2.0](https://spdx.org/licenses/Apache-2.0)
+
+Copyright 2017-2019 Fintech Open Source Foundation
```

Comment:
This copyright notice is incorrect.  [Copyright notices only include the year of first publication](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies) - they are inherently open ended from that year forward.

Reply:
The notice is not incorrect. The Copyright Office's guidance says that a notice should include the year of first publication, not that it should be exclusive of other dates. For materials that are updated regularly over a period of time, it's common and often recommended to include a range of dates.

Reply:
@copiesofcopies [the Copyright Office disagrees...](https://www.copyright.gov/circs/circ03.pdf)

Reply:
@copiesofcopies sure - it doesn't say that Copyright Notices shouldn't include War and Peace either, but I think that not adding spurious elements that are left undefined by the relevant authority is common sense.